### PR TITLE
Import just the tests you want to run

### PR DESCRIPTION
**Pull request by kaapstorm**
otherwise base test classes will be run too.

Do you mean like this, @dannyroberts? Buddy @millerdev.

### CODE REVIEW

File: corehq/apps/locations/tests/__init__.py [hunk 1]
```diff
@@ -1,6 +1,11 @@
 from .test_location_import import *
 from .test_location_types import *
-from .test_locations import *
+from .test_locations import (
```
Comment (dimagimon) [inline corehq/apps/locations/tests/__init__.py]: F401 'LocationGroupTest' imported but unused

Comment (dimagimon) [inline corehq/apps/locations/tests/__init__.py]: F401 'LocationProducts' imported but unused

Comment (dimagimon) [inline corehq/apps/locations/tests/__init__.py]: F401 'LocationsTest' imported but unused

Comment (dimagimon) [inline corehq/apps/locations/tests/__init__.py]: F401 'WrapLocationTest' imported but unused

**Comment by dannyroberts** [comment]
I suppose that works too! I was thinking of making a base WrapIsoFormatTest that somehow never got run (maybe doesn't subclass `TestCase`) and then have all three subclass from it. But this works too!

**Comment by kaapstorm** [comment]
Hmmm. I quite like that idea, because then you wouldn't have to change __init__.py when you add new tests.

**Pull request merged**